layout/css/main.css

```diff
@@ -24,16 +24,20 @@ header img{
 /*Intro*/
 #intro{
 	text-align:center;
-	padding:15px;
+	padding:20px;
 	}
 
-.svgFig {
+
+.svgFig, .svgSpanner {
 	position: relative;
   left: 50%;
   -webkit-transform: translateX(-50%);
   -ms-transform: translateX(-50%);
   transform: translateX(-50%);
 }
+.svgSpanner {
+	min-width: 520px;
```

Comment:
Also a question about media query, 520px is a lot wider than most portrait phones can go so will cause horizontal scrolling